tests/tests/test_integration/test_registration.py

```diff
@@ -45,48 +46,38 @@ def test_registration_niftyreg(tmpdir):
         "--atlas",
         "allen_mouse_100um",
         "-d",
-        data_dir,
+        str(brain_data_dir),
     ]
 
     sys.argv = brainreg_args
     brainreg_run()
+    return test_output_dir
+
 
-    # none of this testing is ideal, as results seem to vary between systems
+@pytest.mark.parametrize(
+    "image",
+    [
+        "boundaries.tiff",
+        "deformation_field_0.tiff",
+        "deformation_field_1.tiff",
+        "deformation_field_2.tiff",
+        "downsampled.tiff",
+        "downsampled_brain data.tiff",
+        "downsampled_standard.tiff",
+        "downsampled_standard_brain data.tiff",
+        "registered_atlas.tiff",
+        "registered_hemispheres.tiff",
+    ],
```

Comment:
Kind of pointless but could do
```suggestion
    [
        "boundaries.tiff",
        "downsampled.tiff",
        "downsampled_brain data.tiff",
        "downsampled_standard.tiff",
        "downsampled_standard_brain data.tiff",
        "registered_atlas.tiff",
        "registered_hemispheres.tiff",
    ] + [f"deformation_field_{i}.tiff" for i in range(3)]
```